packages/docusaurus/src/server/plugins/index.ts

```diff
@@ -47,6 +47,11 @@ export function sortConfig(routeConfigs: RouteConfig[]): void {
       }
     }
 
+    // Root route should get placed last.
+    if (a.path === '/' || b.path === '/') {
```

Comment:
This test is probably good enough, cf snapshot order.

If a !== "/" && b === "/" it should rather return +1

It should work better to do this test at the very beginning of the sort algo instead of the end.

Note if both are "/", we should not return because the route with "/" + subroutes should be the very last